VuFind.js

```diff
@@ -0,0 +1,1804 @@
+				"archive": "ProQuest Dissertations & Theses,",
+				"libraryCatalog": "catalog.lib.uchicago.edu",
+				"numPages": "1",
+				"place": "Ann Arbor",
+				"shortTitle": "Characterizing kinds",
+				"university": "University of Chicago",
+				"attachments": [],
+				"tags": [],
+				"notes": [
+					{
+						"note": "Advisors: Jason Bridges; Christopher Kennedy Committee members: Frank Veltman; Malte Willer"
+					}
+				],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "https://catalog.lib.uchicago.edu/vufind/Record/7143200",
+		"items": [
+			{
+				"itemType": "thesis",
+				"title": "Modern ethical skepticism",
+				"creators": [
+					{
+						"firstName": "Zed",
+						"lastName": "Adams",
+						"creatorType": "author"
+					}
+				],
+				"date": "2008",
+				"callNumber": "BJ2999 Adams",
+				"extra": "OCLC: 232302765",
+				"libraryCatalog": "catalog.lib.uchicago.edu",
+				"numPages": "212",
+				"attachments": [],
+				"tags": [
+					{
+						"tag": "1900-1999"
+					},
+					{
+						"tag": "20th century"
+					},
+					{
+						"tag": "Ethics, Modern"
+					},
+					{
+						"tag": "Ethics, Modern"
+					},
+					{
+						"tag": "Values"
+					},
+					{
+						"tag": "Values"
+					}
+				],
+				"notes": [
+					{
+						"note": "Thesis (Ph. D.)--University of Chicago, Dept. of Philosophy, June 2008"
+					}
+				],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "http://bdtd.ibict.br/vufind/Record/UNICAMP-30_0cff5b4073e71e6635313c7d4e3f205e",
+		"items": [
+			{
+				"itemType": "book",
+				"title": "Criptografia quântica com estados comprimidos da luz",
+				"creators": [
+					{
+						"firstName": "Douglas Delgado de",
+						"lastName": "Souza",
+						"creatorType": "author"
+					}
+				],
+				"date": "2011",
+				"abstractNote": "Orientador: Antonio Vidiella Barranco",
+				"language": "por",
+				"libraryCatalog": "bdtd.ibict.br",
+				"url": "https://hdl.handle.net/20.500.12733/1615696",
+				"attachments": [],
+				"tags": [],
+				"notes": [],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "http://bdtd.ibict.br/vufind/Record/PUC_RIO-1_5e711c01a0c4745a4c27c1750ecfc724",
+		"items": [
+			{
+				"itemType": "book",
+				"title": "A DEPENDENCY TREE ARC FILTER",
```

Comment:
Could we fix all caps in titles & authors? (See e.g. https://github.com/zotero/translators/pull/2876 but we could also consider building a `ZU` function that does this.)